util/src/main/scala/Money.scala

```diff
@@ -194,12 +195,12 @@ case class Money private (amount: BigDecimal, currency: Currency) extends BaseMo
   def partition(ratios: Int*): Seq[Money] = {
     val total = ratios.sum
     val amountInCents = (this.amount / centFactor).toBigInt
-    val amounts = Seq(ratios.map(amountInCents * _ / total):_*)
```

Comment:
I do not understand why it was done this way TBH

Reply:
I cannot understand either.